elements/rh-avatar/rh-avatar.css

```diff
@@ -50,7 +50,7 @@
 }
 
 :host([layout='block']) #container,
-#container.mobile {
```

Comment:
is this related to nav1?

Reply:
It's a fix for avatar that is needed in nav. I can move it to its own PR if that is helpful.

Reply:
's fine